index.yaml

```diff
@@ -3,7 +3,7 @@ entries:
   admission-controller:
   - apiVersion: v2
     appVersion: 0.0.1
-    created: "2022-05-05T22:09:37.397718-07:00"
```

Comment:
Heads up, this is modifying additional timestamps.

Reply:
Thanks! I'm done trying to fight that battle with Helm.